lib/tasks/client_generate.rake

```diff
@@ -39,7 +39,21 @@ class ClientGenerator
   end
 
   def generator_config
-    @generator_config ||= Pathname.new(Rails.root.join(".openapi_generator_config.json")).to_s
+    return @generator_config if @generator_config.present?
+
+    orig_config = Pathname.new(Rails.root.join(".openapi_generator_config.json")).to_s
+    json_settings = File.read(orig_config)
+
+    # Extend config by version used as client's Gem version
+    gem_version = "#{api_version}.0"
+    settings = JSON.parse(json_settings)
+    settings['gemVersion'] = gem_version
+    filepath = "/tmp/openapi_generator_#{gem_version}_#{Time.now.to_i}.json"
```

Comment:
Can you use `Tempfile` instead of a constant path here?  It is possible for two `rake client:generate` tasks (even for two different gems) to overwrite each other's generator_config

Reply:
It's not possible, because Tempfile uses `unlink` method when removed by garbage collector and file is deleted.

Anyway `client:generate` is always run manually by developer, so (s)he probably don't run 2 rake task in the same `Time.now.to_i` time.

Reply:
> It's not possible, because Tempfile uses unlink method when removed by garbage collector and file is deleted.

Isn't that a good thing?  We don't want to leave cruft around in `/tmp` after the rake task finishes.

Reply:
ok, I'll make Tempfile instance variable, it should survive until the system("java...") call finishes